src/common/containers/landing.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import { Anchor } from '../components/button';
+
+import styles from './container.css';
+
+class Landing extends Component {
+  componentWillMount() {
+    if (this.props.auth.authenticated) {
```

Comment:
Agreed. Seeing some public dashboard of another user is different story and probably would likely have different URL scheme from current user dashboard. This could work for example by having: landing/dashboard on `/` (for current user) and `/:user` would be a 'public' dashboard view. So as a logged in user you would see 'edit' view of your dashboard at `/` and any public 'profile' of any user (including you) at `/:user`.

So I'm for having both landing and dashboard at `/` too.
The only reason I see for having them as separate URLs is for auth user to get back to landing page (for example to see some info on it or links to documentation or something).